src/Memory/CLState.ts

```diff
@@ -26,6 +27,7 @@ import { BrowserSlotState } from './BrowserSlot';
  */
 export class CLState {
     private static bbStorage: BB.Storage
+    public static logStorage: ILogStorage
```

Comment:
>ILogStorage [](start = 30, length = 11)

not sure if this should be part of CLState. Shouldn't this be part of CL object?

Reply:
Yea, agree it should not be part of `CLState`.  CLState was for storing current state of bot (messages queue, active model, entities in memory, etc) log dialogs are separate type of storage.

Reply:
done